packages/database/src/migrations/20210610060144-AddFluTrackerLGAPercentNonFirstNationsILI-modifies-data.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+import { deleteObject, generateId, insertObject } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const INDICATOR = {
+  id: generateId(),
+  code: 'FluTracker_LGA_Percent_Non_First_Nations_ILI',
+  builder: 'analyticArithmetic',
+  config: {
+    formula:
+      '(FWV_LGA_004b_Total_By_Country >= 0 and FWV_LGA_003b_Total_By_Country > 0) ? (FWV_LGA_004b_Total_By_Country / FWV_LGA_003b_Total_By_Country * 100) : 0',
```

Comment:
Similar comments here about:
- No need to check numerator is >= 0
- Best not to multiply by 100
- Should be able to unpack this indicator to not use indicator parameters 👍